src/omni-slider.js

```diff
@@ -1,532 +1,540 @@
-/* Constructor function
- * elementContainer - this acts as a wrapper for the slider, all of the sliders DOM elements will be transcluded inside the <element> provided
- * options - contains the options for the slider
- */
-function Slider(elementContainer, options) {
+(function() {
```

Comment:
Minor question: Do you really need IIFE and `'use strict';` ?

I was able to comment out the outer wrapper of IIFE & strict mode code, could still able to run `npm start` to edit changes and build the correct artifacts, as the highlighted IDE files in the 1st photo.

Then, I was able to verify that the Demo page in the browser then the build artifacts without IIFE & strict mode could run correctly, without any run-time console error, as in 2nd photo.

<img width="1280" alt="screen shot 2016-06-05 at 8 21 11 am" src="https://cloud.githubusercontent.com/assets/19755488/15805404/387cfbf4-2af7-11e6-86d0-17e64d31601f.png">

<img width="1048" alt="screen shot 2016-06-05 at 8 17 03 am" src="https://cloud.githubusercontent.com/assets/19755488/15805405/401b857e-2af7-11e6-9354-a667c1973bea.png">